src/js/components/LoadFilesInput.js

```diff
@@ -37,6 +37,7 @@ export default function LoadFilesInput({onChange}: Props) {
       {...bindDropzone()}
     >
       <input
+        tabIndex="-1"
```

Comment:
Makes this not tabbable since it's hidden.

Reply:
neat trick